common/source/docs/common-esp32-telemetry.rst

```diff
@@ -0,0 +1,210 @@
+  NodeMCU style DevKit Boards with an IPEX port for an external antenna
+  often also offer an onboard antenna that is activated by default. You
+  may need to resolder a resistor to activate the external antenna port.
+
+Downloading and Flashing the Firmware
+-------------------------------------
+
+`Download the firmware from the GitHub repository`_ and `follow the
+flashing instructions there`_. They are always up to date.
+
+.. note::
+
+  `Follow the flashing instructions inside the GitHub Repository.`_ The
+  exact parameters may differ from release to release of DroneBridge for
+  ESP32.
+
+For convenience reasons some short instructions are given here:
+
+-  `Download the pre-compiled firmware binaries`_
+-  Connect your DEVKit to your computer via USB/Serial bridge (most
+   DevKits already offer a USB port for flashing and debugging)
+-  Erase the flash and flash the DroneBridge for ESP32 firmware onto
+   your ESP32
+
+   -  Using `Espressif Flash Download Tool`_ (Windows only)
+   -  Using esp-idf/esptool (all platforms)
+
+-  Power Cycle the ESP32
+-  Connect to the "DroneBridge for ESP32" WiFi network and configure
+   the firmware for your application
+
+Configuring DroneBridge for ESP32
+---------------------------------
+
+You can change the default configuration via the Webinterface.
+Connect to the ESP32 via WiFi and enter ``dronebridge.local``, ``http://dronebridge.local`` or ``192.168.2.1`` in the address
+bar of your browser.
+
+Default Configuration
+~~~~~~~~~~~~~~~~~~~~~
+
+-  SSID: ``DroneBridge for ESP32``
+-  Password: ``dronebridge``
+-  Transparent/MAVLink
+-  UART baud rate ``115200``
+-  UART TX pin ``17``
+-  UART RX pin ``16``
+-  Gateway IP: ``192.168.2.1``
+
+Custom Settings & Webinterface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+| You can change the default configuration via the Webinterface.
+| Connect to the ESP32 via WiFi and enter ``dronebridge.local``,
+  ``http://dronebridge.local`` or ``192.168.2.1`` in the address bar of
+  your browser.
+
+.. image:: https://raw.githubusercontent.com/DroneBridge/ESP32/master/wiki/dbesp32_webinterface.png
+   :alt: DroneBridge for ESP32 Webinterface
+
+.. note::
+
+  Some settings require you to reboot the ESP32 to take effect.
+
+Wiring
+======
+
+Wiring is very simple and mostly the same for all devices connected to
+the TELEM1/2 ports of a Pixhawk. That is why this guide does not go into
+detail here.
+
+-  Connect UART of ESP32 to a UART of your flight controller (e.g. TELEM
+   1 or TELEM 2 port). Make sure the voltage levels match! Most ESP32
+   DevKits can only take 3.3V!
+
+   -  TX to RX
+   -  RX to TX
+   -  GND to GND
+   -  Stable 3.3V or 5V power supply to the ESP32 (depending on the
+      available inputs of your DevKit)
+
+-  Set the flight controller port to the desired protocol.
+
+Some manufacturers of ESP32 DevKits have wrong labels for the
+pins on their products. Make sure that the PINs on your board are
+labeled correctly if you encounter issues.
+
```

Comment:
Need autopilot configuration section

Reply:
updated

Reply:
You mean Mission Planner config to work with the ESP?

Reply:
no, for the autopilot UART port to work with it...correct protocol and baud rate must be setup on desired port

Reply:
Added